src/raygun.rum/index.js

```diff
@@ -1055,7 +1068,7 @@ var raygunRumFactory = function (window, $, Raygun) {
        */
       var duration = timing.duration;
 
-      if (duration !== 0) {
+      if (duration !== 0 || (timing.responseEnd === undefined || timing.startTime === undefined)) { //Stops potential call to deprecated VisibilityStateEntry.responseEnd
```

Comment:
simplification could be made here 
```suggestion
      if (duration !== 0 || (!timing.responseEnd || !timing.startTime)) {
```

Reply:
Moving the `!timing.responseEnd || !timing.startTime` check out into a variable above, ln:1070~ would make it easier to understand why we're making the check.

eg (the var name could be better)
```
var mitigateDepricatedApi = !timing.responseEnd || !timing.startTime;
```

Reply:
This is a great idea! ill add it in

Reply:
For visibility I changed the check to an && as || was not producing the right outcomes